cli/test/support/testData.ts

```diff
@@ -0,0 +1,85 @@
+import { csvw, hydra, rdf, schema } from '@tpluscode/rdf-ns-builders'
+import { cc } from '@cube-creator/core/namespace'
+import { DELETE, INSERT } from '@tpluscode/sparql-builder'
+import StreamClient from 'sparql-http-client/StreamClient'
+import env from '@cube-creator/core/env'
+
+export const client = new StreamClient({
+  updateUrl: 'http://db.cube-creator.lndo.site/cube-creator/update',
+  endpointUrl: 'http://db.cube-creator.lndo.site/cube-creator/query',
+})
+
+const clientOptions = () => ({
+  base: env.API_CORE_BASE,
+})
+
+function removeTestGraphs() {
+  return DELETE`graph ?g { ?s ?p ?o }`
+    .WHERE`VALUES ?g {
+      </project/cli-test>
+      </project/cli-test/mapping>
+      </project/cli-test/tables>
+      </project/cli-test/mapping/table/foo>
+      </project/cli-test/source/foo>
+    }`
+    .WHERE`graph ?g { ?s ?p ?o }`
+    .execute(client.query, clientOptions())
+}
+
+export const insertTestData = async () => {
+  await removeTestGraphs()
+
+  return INSERT.DATA`
+  graph </project/cli-test> {
+      </project/cli-test>
+          a ${hydra.Resource} , ${cc.CubeProject} ;
+          ${cc.csvMapping} </project/cli-test/mapping> ;
+          ${cc.cube}       </cube/cli-test>
```

Comment:
I assume a `cc:cube` which would be the actual instance of project's `qb:Cube`

Now, currently observations will be pushed into its graph but I reconsider and I would propose to have them separated

```sparql
graph <project/foo/cube> {
  <project/foo/cube> a qb:Cube .
}

graph <project/foo/cube/observation> {
   # pipeline output here
}
```

This way we can easily reject the previous transformation result but keep the cube itself